model_analyzer/plots/plot.py

```diff
@@ -158,4 +159,4 @@ def save(self, filepath):
             this plot should be saved to
         """
 
-        self._fig.savefig(os.path.join(filepath, self._name))
+        self._fig.savefig(os.path.join(filepath, self._name), dpi=100, bbox_inches='tight')
```

Comment:
What exactly is this doing?

Reply:
It is a 'trick' that I remember from grad school days to make the plots look good. Second argument make sure that the subplot fits the figure and the other argument increases the quality of image.